R/parametric-measures.R

```diff
@@ -566,29 +566,29 @@ calc_SMD <- function(A_data, B_data,
 
 # Calculate statistics to be used in calculating log ratio of medians
 
-stats_LRM <- function(data, warn = TRUE) {
+stats_LRM <- function(data, delta_method = FALSE, warn = TRUE) {
   
   n <- length(data)
-  median <- median(data)
-  o1 <- round(n/2 - sqrt(n))
-  if (o1 < 1) {o1 = 1}
+  median_est <- median(data)

```

Comment:
If median of baseline data or treatment phase data is 0, the LRM estimate would be inf. Should we do some truncation and use the truncated means for LRR (Pustejovsky, 2018) with 1/(2Dm) for baseline median and 1/(2Dn) for treatment median?

Reply:
Yes that’d be good, although it does add a lot of extra arguments.